lib/mix/lib/mix/tasks/format.ex

```diff
@@ -36,18 +36,26 @@ defmodule Mix.Tasks.Format do
   If any of the `--check-*` flags are given and a check fails, the formatted
   contents won't be written to disk nor printed to stdout.
 
-  ## .formatter.exs
+  ## `.formatter.exs`
 
   The formatter will read a `.formatter.exs` in the current directory for
   formatter configuration. It should return a keyword list with any of the
   options supported by `Code.format_string!/2`.
 
-  The `.formatter.exs` also supports an `:inputs` field which specifies the
-  default inputs to be used by this task:
+  The `.formatter.exs` also supports other options:
 
-      [
-        inputs: ["mix.exs", "{config,lib,test}/**/*.{ex,exs}"]
-      ]
+    * `:input` (a list of paths and patterns) - specifies the default inputs
+      to be used by this task. For example, `["mix.exs", "{config,lib,test}/**/*.{ex,exs}"]`.
+
+    * `:import_deps` (a list of dependencies as atoms) - specifies a list
+       of dependencies whose formatter configuration will be imported.
+       See the "Importing dependencies configuration" section below for more
+       information.
+
+    * `:export_locals_without_parens` (a list of function names and arities) -
```

Comment:
I was thinking an `:export` keys with a `:locals_without_parens` inside. But this way also works. What do you prefer?

Reply:
I like them both. Maybe `:export` with subkeys is more flexible for the future. Shall we go with that?

Reply:
:+1:.